addons/website/static/src/scss/website.scss

```diff
@@ -232,6 +232,7 @@ font[class*='bg-'] {
 .oe_img_bg {
     background-size: cover;
     background-repeat: no-repeat;
+    background-position: 50% 50%;
```

Comment:
@bvr-odoo No, add a new class which will handle that and add it by default on snippets and for the JS code which adds the oe_img_bg class. That way it will be compatible... at the cost of one extra CSS class.